test-data/unit/fine-grained-modules.test

```diff
@@ -2158,23 +2158,6 @@ main.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mis
 main.py:2: error: Incompatible types in assignment (expression has type "int", variable has type "str")
 ==
 
-[case testMissingStubAdded2]
-# flags: --follow-imports=skip --py2
```

Comment:
This test case might still be relevant. I think that it's better to just drop the `--py2` flag but keep the test case.